esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -2096,13 +2096,11 @@ void CWsDfuEx::doGetFileDetails(IEspContext &context, IUserDescriptor *udesc, co
             IPropertyTree &tree = itr->query();
             const char *owner = tree.queryProp("@name");
             const char *modified = tree.queryProp("@modified");
-            int count = tree.getPropInt("@count", 0);
             Owned<IEspDFUFileProtect> protect= createDFUFileProtect();
             if(owner && *owner)
                 protect->setOwner(owner);
             if(modified && *modified)
                 protect->setModified(modified);
-            protect->setCount(count);
```

Comment:
if (version < 1.54)
    protect->setCount(tree.getPropInt("@count", 0));

Reply:
I think should probably be:
```
if (version < 1.54)
    protect->setCount(1);
```

.. since it didn't have any sensible meaning when it was >1 before, and had to be >=1 if present at all.

Reply:
OK for me.